src/Collections/PropertyCollection.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+
+namespace ArmadilloEstates\Collections;
+
+
+use ArmadilloEstates\Entities\PropertyEntity;
+
+class PropertyCollection
+{
+    private $allProperties;
+
+    public function __construct(array $properties)
+    {
+        $this->propertyCheck($properties);
+        $this->allProperties = $properties;
+    }
+
+    private function propertyCheck($properties)
+    {
+        foreach ($properties as $property) {
+            if (!($property instanceof PropertyEntity)) {
+                throw new \Exception('Array must only contain instances of PropertyEntity');
```

Comment:
What does the \ do here ?
can it be 
```suggestion
                throw new Exception('Array must only contain instances of PropertyEntity');
```

Reply:
ohh it is related to namesapce!?